core/components/atoms/_overlay/overlay.js

```diff
@@ -21,6 +28,12 @@ class Overlay extends React.Component {
     this.contentElement = null
   }
 
+  static getSizeForOverlay(propValue) {
+    if (typeof propValue === 'number') return `${propValue}px`
```

Comment:
Do we want to support an open API for this?

Reply:
Not for now